frame/examples/basic/src/benchmarking.rs

```diff
@@ -34,25 +34,25 @@ use frame_system::RawOrigin;
 // Details on using the benchmarks macro can be seen at:
 //   https://paritytech.github.io/substrate/master/frame_benchmarking/trait.Benchmarking.html#tymethod.benchmarks
 benchmarks! {
-	// This will measure the execution time of `set_dummy` for b in [0..1000] range.
+	// This will measure the execution time of `set_dummy`.
 	set_dummy_benchmark {
-		// This is the benchmark setup phase
-		let b in 0 .. 1000;
-	}: set_dummy(RawOrigin::Root, b.into()) // The execution phase is just running `set_dummy` extrinsic call
+		// This is the benchmark setup phase.
+		let value = 1000u32.into();
```

Comment:
Since this is supposed to be an example maybe it'd be a good idea to explicitly say that "the `set_dummy` always takes constant time hence we're hardcoding the `value` here" or something like that?